### ISSUE TITLE
Memory Leak

### ISSUE THREAD

**Issue by vanniktech** [palaima/DebugDrawer#9]
I've just started implementing [LeakCanary](https://github.com/square/leakcanary) in our application and I've found that this library leaks the Context.

![screen shot 2015-06-18 at 10 52 36 am](https://cloud.githubusercontent.com/assets/5759366/8227502/4a794fc0-15a8-11e5-93bf-10726bb5f596.png)

**Comment by balysv** [comment]
This is likely due to `WifiManager` initialization in `NetworkController` using the `Activity` but not the `Application` `Context` https://github.com/palaima/DebugDrawer/blob/master/debugdrawer/src/main/java/io/palaima/debugdrawer/controller/NetworkController.java#L69

Should be fixed by

```java
mWifiManager = (WifiManager) mContext.getSystemService(Context.WIFI_SERVICE);
mConnectivityManager = (ConnectivityManager) mContext.getSystemService(Context.CONNECTIVITY_SERVICE);
```

The library doesn't compile at the moment, will test when it's working.

**Comment by vanniktech** [comment]
@balysv 
I've made two PR one for LeakCanary itself and one for the missing resource.

**Comment by palaima** [comment]
released v0.3.1
